sale_order_line_date/models/sale_order_line.py

```diff
@@ -12,9 +12,8 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    commitment_date = fields.Datetime(oldname="requested_date")
+    commitment_date = fields.Datetime()
```

Comment:
```suggestion
    commitment_date = fields.Datetime('Delivery Date')
```
Like in sale orders.